ci-operator/step-registry/openshift-extended/test/openshift-extended-test-commands.sh

```diff
@@ -224,6 +224,30 @@ oc wait nodes --all --for=condition=Ready=true --timeout=15m
 oc wait clusteroperators --all --for=condition=Progressing=false --timeout=15m
 oc get clusterversion version -o yaml || true
 
+# Get the KUBEADMIN_REMOVED environment variable (or set a default false if missing)
+KUBEADMIN_REMOVED="${KUBEADMIN_REMOVED:-false}"
+# Function to delete the kubeadmin secret if KUBEADMIN_REMOVED is "true"
+function remove_admin_user() {
+  if [[ "$KUBEADMIN_REMOVED" == "true" ]]; then
+    echo "KUBEADMIN_REMOVED is set to 'true'. Deleting kubeadmin secret..."
+    oc delete secrets kubeadmin -n kube-system
+
+    if test -f "${SHARED_DIR}/nested_kubeconfig"; then
+      echo "KUBEADMIN_REMOVED is set to 'true'. Deleting kubeadmin for GUEST Cluster"
+      HOSTED_CLUSTER_NAME=$(oc get hostedclusters -n clusters -o jsonpath='{.items[0].metadata.name}')
```

Comment:
I'm not sure if `[0]` is robust in case it happens to have multiple HCs. Need consult HyperShift subteam if it is possible to infer HC from ${SHARED_DIR}/nested_kubeconfig or other info.

Reply:
Adding logic suggested by @kuiwang02